codalab/worker/worker.py

```diff
@@ -275,7 +275,7 @@ def start(self):
                     # We sleep in 5-second increments to check
                     # if the worker needs to terminate (say, if it's received
                     # a SIGTERM signal).
-                    logger.error('Sleeping for 1 hour due to exception...please help me!')
+                    logger.warning('Sleeping for 1 hour due to exception...please help me!')
```

Comment:
Is this an innocuous message?

Reply:
i think so, because we've already printed the `Exception` traceback just before it (`traceback.print_exc()`). The message is just conveying that the worker is sleeping for an hour.

Reply:
I'd imagine that we'd want to record this error in sentry, though. Would this mean that Sentry isn't recording this though? or would the previous `traceback.print_exc()` function also get logged in Sentry?

Reply:
the issue is that logging this message isn't actually informative of what's happening. The current behavior with sentry (which i've actually really enjoyed using, fwiw) means that an exception gets logged 3 times:
- The actual exception is logged. This is the most important, since it also provides the stack trace / code context / what the values of local variables were, which is enormously useful for debugging
- The traceback is logged as text (I think from `traceback.print_exc()`).
- A message `Sleeping for 1 hour due to exception...please help me!` / `'Encountered exception, terminating the worker after sleeping for 5 minutes...'` is logged, which is the least informative of the bunch.